test/test_helper.rb

```diff
@@ -0,0 +1,33 @@
+ENV['RAILS_ENV'] ||= 'test'
+require_relative '../config/environment'
+require 'rails/test_help'
+require "minitest/rails"
+require "minitest/reporters"  # for Colorized output
+#  For colorful output!
+Minitest::Reporters.use!(
+  Minitest::Reporters::SpecReporter.new,
+  ENV,
+  Minitest.backtrace_filter
+)
+
+class ActiveSupport::TestCase
+  # Run tests in parallel with specified workers
+  # parallelize(workers: :number_of_processors) # causes out of order output.
+
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  fixtures :all
+
+  def login(name = "Tina")
+    user_hash = {
+      user: {
+        name: "Tina"
```

Comment:
I think you mean to use `name` here instead of hard coding `"Tina"`:

```suggestion
        name: name
```